src/DataDepsGenerators.jl

```diff
@@ -97,9 +141,9 @@ function format_checksums(csum::AbstractString)
     if length(csum)>0 "\"$csum\"" else "" end
 end
 
-function format_checksums(::Void)
-    ""
-end
+format_checksums(::Void) = missing
```

Comment:
delete `format_checksums(::Void)`,
becomes things that are currently returning `nothing` for checksums should be returning `missing`.
Deleting this will help us find them